### DISCUSSION TITLE
Welcome to dynotxt Discussions!

### DISCUSSION THREAD

**Discussion by remintroy** [remintroy/dynotxt#125] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by maheswar-dileep** [comment]
I am interested in contributing to this project. 💯

**Reply by remintroy** [reply]
Wow that's great to hear 👍🏻
